packages/cf-style-const/src/variables.js

```diff
@@ -14,6 +14,20 @@ export default {
     seven: 'linear-gradient(left, #F69259, #FFDB6F)',
     eight: 'linear-gradient(left, #85CBA8, #FFDB6F)'
   },
+  color: {
+    rain: '#408BC9',
```

Comment:
These  names were created by designers. @jculvey likes it too. I don't care that much if it is consistent between design and engineering. I'm already using these colors in cf-ux.